### PR TITLE
Adding the Cloud Community Days online conference.

### PR DESCRIPTION
**Pull request by dharmendranegi**
**Details of the conference**:

- Website: https://ccdays.konfhub.com/
- Date:  17-19 June 2020 
- Venue:  Online
- Description: Ride the Cloud Revolution

### CODE REVIEW

File: year-pages/2020.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
+| [Cloud Community Days](https://ccdays.konfhub.com/) | 17-19 Jun | Online | Ride the Cloud Revolution | No |
```
Comment (nikhita) [inline year-pages/2020.md:5]:
```suggestion
| [Cloud Community Days](https://ccdays.konfhub.com/) | 17-19 Jun | Online | Ride the wave of latest and best in Containers, Serverless, AI & ML, and more. | No |
```

Comment (dharmendranegi) [reply year-pages/2020.md]: okay, i will do.

Comment (nikhita) [reply year-pages/2020.md]: @dharmendranegi I already made the changes (turns out GitHub allows maintainers to do that!) and the code has been merged. :)

**Comment by nikhita** [comment]
Woops, this should have been in the README.md file. Can you please send another to add this to the README?

File: year-pages/2020.md (hunk 1 above)
Comment (dharmendranegi) [reply year-pages/2020.md]: Thanks.

**Comment by dharmendranegi** [comment]
okay

**Pull request merged**